main/src/main/scala/org/clulab/reach/darpa/HypothesisHandler.scala

```diff
@@ -78,7 +78,9 @@ object HypothesisHandler {
       case event:BioEventMention =>
 
         // Get the dependencies of the sentence
-        val dependencies = event.sentenceObj.dependencies.getOrElse(new DirectedGraph[String](Nil, Set[Int]()))
+        // TODO: Keith/Mihai, could you verify if this signature change is appropriate?
+        val dependencies = event.sentenceObj.dependencies.getOrElse(new DirectedGraph[String](Nil, None))
```

Comment:
It might depend on which version of processors this will be using.  In the current master branch, I believe it is OK and 8.5.0 looks good as well.  The None should not be needed because there's a default value configured.